src/System.IO.Compression.Brotli/System/IO/Compression/Brotli/BrotliPrimitives.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+#if BIT64
+    using nuint = System.UInt64;
+#else
+    using nuint = System.UInt32;
+#endif 
+
+namespace System.IO.Compression
+{
+    public struct BrotliPrimitives
+    {
+        const int defQuality = 11;
+        const int defLgWin = 24;
+
+        public static bool Compress(ReadOnlySpan<byte> source, Span<byte> destination, out nuint bytesConsumed, out nuint bytesWritten)
```

Comment:
Yes, they should be nuint, because 1. array size can't be negative(so number of bytes in byte array also). 2. the native signature is: `size_t input_size,
    const uint8_t* input_buffer, size_t* encoded_size,
    uint8_t* encoded_buffer` ans size_t is unsigned type.